src/services/dashboards/dashboard-customize/modules/DashboardAddWidgetModal.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+    <!--song-lang-->
+    <p-button-modal :visible.sync="proxyVisible"
+                    :header-title="$t('Add New Widget')"
+                    size="lg"
+                    class="dashboard-add-widget-modal"
+    >
+        <template #body>
+            <p-tab :tabs="tabState.tabs"
+                   :active-tab.sync="tabState.activeTab"
+            >
+                <template #default-widget>
+                    <dashboard-default-widget-tab />
+                </template>
+            </p-tab>
+        </template>
+    </p-button-modal>
+</template>
+<script lang="ts">
+import type { SetupContext } from 'vue';
+import {
+    computed, defineComponent, reactive, toRefs,
+} from 'vue';
+
+import { PButtonModal, PTab } from '@spaceone/design-system';
+import type { TabItem } from '@spaceone/design-system/src/navigation/tabs/tab/type';
+
+import { i18n } from '@/translations';
+
+import { useProxyValue } from '@/common/composables/proxy-state';
+
+import DashboardDefaultWidgetTab from '@/services/dashboards/dashboard-customize/modules/DashboardDefaultWidgetTab.vue';
+
+interface Props {
+    visible: boolean;
+}
+export default defineComponent<Props>({
+    name: 'DashboardAddWidgetModal',
+    components: {
+        DashboardDefaultWidgetTab,
+        PTab,
+        PButtonModal,
+    },
+    props: {
+        visible: {
+            type: Boolean,
```

Comment:
there is no `default` value. Is it fine??

Reply:
Oh I missed it! thanks!